examples/raspi/src/main.rs

```diff
@@ -0,0 +1,122 @@
+use std::error::Error;
+use std::f32::consts::PI;
+use std::{env, sync::Arc, thread, time::Duration};
+
+use rppal::pwm::{Channel, Polarity, Pwm};
+use mavlink::error::MessageReadError;
+use mavlink::common::MavMessage;
+use mavlink;
+
+// Servo configuration. The datasheet was wrong for mine so these are manually calibrated.
+// Your configuration may be different.
+const PERIOD_MS: u64 = 20;
+const PULSE_MIN_US: u64 = 640;
+const PULSE_NEUTRAL_US: u64 = 1540;
+const PULSE_MAX_US: u64 = 2540;
```

Comment:
I think we should provide a more generic and compatible example rather than telling people to adjust these values to fit their system.